servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -214,48 +306,167 @@ public Duration maxEjectionTimeJitter() {
     }
 
     /**
-     * A builder for {@link OutlierDetectorConfig} instances.
+     * Make a {@link Builder} that is pre-populated with the values of this config object.
+     * @return a {@link Builder} that is pre-populated with the values of this config object.
      */
-    public static final class Builder {
-        private Duration ewmaHalfLife = Duration.ofSeconds(10);
-        private int consecutive5xx = 5;
-
-        private Duration interval = Duration.ofSeconds(10);
-
-        private Duration baseEjectionTime = Duration.ofSeconds(30);
-
-        private int maxEjectionPercentage = 10;
-
-        private int enforcingConsecutive5xx = 100;
-
-        private int enforcingSuccessRate = 100;
-
-        private int successRateMinimumHosts = 5;
-
-        private int successRateRequestVolume = 100;
-
-        private int successRateStdevFactor = 1900;
-
-        private int failurePercentageThreshold = 85;
-
-        private int enforcingFailurePercentage;
+    public Builder toBuilder() {
```

Comment:
We don't have this pattern anywhere else. Consider either always starting with a default `new Builder()` or if you need a way to copy pre-existing configuration it could be something like `new Builder(OutlierDetectorConfig)`. This seems necessary only for testing, users unlikely need a cope of pre-existing detector config. They can just reuse a builder to create multiple configs if necessary.

Reply:
I'll go with the `new Builder(OutlierDetectorConfig)` pattern, which does exist at least internally. We can keep it package private for now as well.